packages/docs/getting-started/running-a-full-node-in-rc1.md

```diff
@@ -0,0 +1,111 @@
+{% endhint %}
+
+## Celo Networks
+
+First we are going to setup the environment variables required for `Baklava` network. Run:
+
+```bash
+export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:rc1
+export NETWORK_ID=42220
+```
+
+## Pull the Celo Docker image
+
+We're going to use a Docker image containing the Celo node software in this tutorial.
+
+If you are re-running these instructions, the Celo Docker image may have been updated, and it's important to get the latest version.
+
+```bash
+docker pull $CELO_IMAGE
+```
+
+## Set up a data directory
+
+First, create the directory that will store your node's configuration and its copy of the blockchain. This directory can be named anything you'd like, but here's a default you can use. The commands below create a directory and then navigate into it. The rest of the steps assume you are running the commands from inside this directory.
+
+```bash
+mkdir celo-data-dir
+cd celo-data-dir
+```
+
+## Create an account and get its address
+
+In this step, you'll create an account on the network. If you've already done this and have an account address, you can skip this and move on to configuring your node.
+
+Run the command to create a new account:
+
+```bash
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE account new
+```
+
+It will prompt you for a passphrase, ask you to confirm it, and then will output your account address: `Address: {<YOUR-ACCOUNT-ADDRESS>}`
+
+Save this address to an environment variables, so that you can reference it below (don't include the braces):
+
+```bash
+export CELO_ACCOUNT_ADDRESS=<YOUR-ACCOUNT-ADDRESS>
+```
+
+{% hint style="info" %}
+This environment variable will only persist while you have this terminal window open. If you want this environment variable to be available in the future, you can add it to your `~/.bash_profile`
+{% endhint %}
+
+## Configure the node
+
+The genesis block is the first block in the chain, and is specific to each network. This command gets the `genesis.json` file for baklava and uses it to initialize your nodes' data directory.
+
+```bash
+docker run -v $PWD:/root/.celo --rm $CELO_IMAGE init /celo/genesis.json
+```
+
+In order to allow the node to sync with the network, get the enode URLs of the bootnodes:
+
+```bash
+export BOOTNODE_ENODES=$(docker run --rm --entrypoint cat $CELO_IMAGE /celo/bootnodes)
+```
+
+## Start the node
+
+This command specifies the settings needed to run the node, and gets it started.
+
+```bash
+docker run --name celo-fullnode -d --restart unless-stopped -p 127.0.0.1:8545:8545 -p 127.0.0.1:8546:8546 -p 30303:30303 -p 30303:30303/udp -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --networkid $NETWORK_ID --syncmode full --rpc --rpcaddr 0.0.0.0 --rpcapi eth,net,web3,debug,admin,personal --light.serve 90 --light.maxpeers 1000 --maxpeers 1100 --etherbase $CELO_ACCOUNT_ADDRESS --bootnodes $BOOTNODE_ENODES --nousb
```

Comment:
do we need to include `--gcmode=archive`?

Reply:
also, would suggest adding a hint that allows you to run with Ledger :)

Reply:
You shouldn't need to run an archive node